README.md

```diff
@@ -1,2 +1,27 @@
-# community
-Salt Extensions Community Repository for Meeting Notes, Docs, etc.
+# Salt Extensions working group
```

Comment:
**nitpick (non-blocking):** does it make sense to use "Salt Extensions Community" as the H1 README header? the contents of this repository are not necessarily relegated only to working group activities. then we could have the working group section as an H2 below it.

Reply:
I can see your point! I'll make that change.